config/package_resource.go

```diff
@@ -126,60 +127,70 @@ func getPackageInfoCacheKey(pkgFile *agentendpointpb.OSPolicy_Resource_File) (st
 	return base64.RawStdEncoding.EncodeToString(raw), nil
 }
 
-func getPackageInfoCache() packageInfoCache {
+func loadPackageInfoCache(ctx context.Context) {
+	if packageInfoCacheStore != nil {
+		return
+	}
 	data, err := ioutil.ReadFile(packageInfoCacheFile)
 	if err != nil {
 		// Just ignore the error and return early
-		// The error mode here is just always redownload the file.
-		return nil
+		// The error mode here is to just always redownload the file.
+		clog.Debugf(ctx, "Error reading the package info cache: %v", err)
```

Comment:
Should these be warnings?

Reply:
I dont want spam, users can't solve the issue so I want the logs to be very minimal.